src/modules/pythonmonkey/pythonmonkey.cc

```diff
@@ -174,20 +174,17 @@ static bool getEvalOption(PyObject *evalOptions, const char *optionName, bool *b
   PyObject *value;
 
   value = PyDict_GetItemString(evalOptions, optionName);
-  if (value) {
-    if (PyLong_Check(value))
-      *b_p = PyBool_FromLong(PyLong_AsLong(value));
-    else
-      *b_p = value != Py_False;
-  }
+  if (value)
+    *b_p = PyObject_IsTrue(value) ? true : false;
   return value != NULL;
 }
 
 static PyObject *eval(PyObject *self, PyObject *args) {
+  size_t argc = PyTuple_GET_SIZE(args);
   StrType *code = new StrType(PyTuple_GetItem(args, 0));
```

Comment:
```suggestion
  StrType *code = argc > 0 ? new StrType(PyTuple_GetItem(args, 0)) : nullptr;
```
https://docs.python.org/3/c-api/tuple.html#c.PyTuple_GetItem
`PyTuple_GetItem` returns `NULL` and sets an [IndexError](https://docs.python.org/3/library/exceptions.html#IndexError) exception if index out of bounds, although the exception is later overridden by `PyErr_SetString(PyExc_TypeError, ...`

Reply:
This is fixed a different way in `3011a812fa7ca367ffe277a7a7c74dfff9cd971b` - not applying this to save us the merge conflict later